### ISSUE TITLE
UISP - Command Line Interface (CLI)

### ISSUE THREAD

**Issue by Hartuty** [Nico640/docker-unms#114]
Is there a way i can run uisp cli from the container?

Kind regards

**Comment by schneini** [comment]
No, this image does not have an unms-cli command, because it's actually only a bash script that executes a command in the appropriate container via docker compose. But because everything is contained in a single container here, the unms-cli script wouldn't work.

However, you can view the unms-cli script and check what it's actually executing inside the container and then simply execute that yourself. What do you think you need unms-cli for?

**Comment by Nico640** [comment]
No, this image does not have an unms-cli command, because it's actually only a bash script that executes a command in the appropriate container via docker compose. But because everything is contained in a single container here, the unms-cli script wouldn't work.

However, you can look inside the unms-cli script and check what it's actually executing inside the container and then simply execute that yourself. What do you need unms-cli for?

**Comment by tobiasvoglhuber** [comment]
If you want to migrate from UISP cloud to a local/self hosted instance you need change user credencials via the the uisp-cli after you have imported the backup or you can't log in.
https://help.ui.com/hc/en-us/articles/23788682058135-UISP-Cloud-Hosting-Migration-scenarios#16

**Comment by Nico640** [comment]
Yes, for that the following commands can be executed inside the container:
- List users: `node cli/user.js password`
- Change user password: `node cli/user.js password USERNAME` (replace USERNAME with the username of the UISP user, it will prompt for a new password)

**Comment by Hartuty** [comment]
I wanted to change the credentials after restoring a backup as explained by @tobiasvoglhuber.
But it has worked from the container as per the above commands
